src/main/resources/eventsFiltersSection.json

```diff
@@ -0,0 +1,25 @@
+{
+  "page": {
+    "title": "Filter By"
+  },
+  "filters": {
+    "type": [
+      "IN_PERSON",
+      "ONLINE_MEETUP"
+    ],
+    "topics": [
+      "Book Club",
+      "Writing Club",
+      "Tech Talk"
+    ],
+    "date": [
+      "IN_30_DAYS",
+      "IN_30_TO_60_DAYS",
+      "MORE_THAN_60_DAYS"
+    ],
+    "region": [
+      "London",
+      "Spain"
```

Comment:
I think so far we don't have any other city, so maybe just return "London".

Reply:
ok will make change to the json